src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -0,0 +1,464 @@
+        :math:`2 \times 2`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    params : (float), optional
+        Parameter :math:`\mu` for the Van der Pol Equations
+        Default is :math:`\mu=0.1`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing the Van der Pol Oscillator IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return vanderpol_rhs(t, y, params)
+
+    def jac(t, y):
+        return vanderpol_jac(t, y, params)
+
+    return IVP(timespan, initrv, rhs, jac=jac)
+
+
+def vanderpol_rhs(t, y, params):
+    y1, y2 = y
+    if isinstance(params, float):
+        mu = params
+    else:
+        (mu,) = params
+
+    return np.array([y2, mu * (1.0 - y1 ** 2) * y2 - y1])
+
+
+def vanderpol_jac(t, y, params):
+    y1, y2 = y
+    if isinstance(params, float):
+        mu = params
+    else:
+        (mu,) = params
+
+    return np.array([[0.0, 1.0], [-2.0 * mu * y2 * y1 - 1.0, mu * (1.0 - y1 ** 2)]])
+
+
+def threebody(timespan, initrv, params=0.012277471):
+    r"""Initial value problem (IVP) based on a three-body problem.
+
+    The three-body problem is defined as follows:
+    Let the initial conditions be :math:`y = (y_1, y_2, \dot{y}_1, \dot{y}_2)^T`.
+
+    .. math::
+
+        f(t, y) =
+        \begin{pmatrix}
+            \dot{y_1} \\
+            \dot{y_2} \\
+            y_1 + 2 \dot{y}_2 - \frac{(1 - \mu) (y_1 + \mu)}{d_1}
+                - \frac{\mu (y_1 - (1 - \mu))}{d_2} \\
+            y_2 - 2 \dot{y}_1 - \frac{(1 - \mu) y_2}{d_1} - \frac{\mu y_2}{d_2}
+        \end{pmatrix}
+
+    with
+
+    .. math::
+
+        d_1 &= ((y_1 + \mu)^2 + y_2^2)^{\frac{3}{2}} \\
+        d_2 &= ((y_1 - (1 - \mu))^2 + y_2^2)^{\frac{3}{2}}
+
+    and a constant parameter  :math:`\mu` denoting the standardized moon mass.
+    Default is :math:`\mu = 0.012277471`.
+
+    Parameters
+    ----------
+    timespan : (float, float)
+        Time span of IVP.
+    initrv : RandomVariable,
+        *(shape=(4, ))* -- Vector-valued RandomVariable that describes the belief
+        over the initial value. Usually its distribution is Constant (noise-free)
+        or Normal (noisy) with :math:`4`-dimensional mean vector and
+        :math:`4 \times 4`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    params : (float), optional
+        Parameter :math:`\mu` for the three-body problem
+        Default is :math:`\mu = 0.012277471`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing a three-body problem IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return threebody_rhs(t, y, params)
+
+    return IVP(timespan, initrv, rhs)
+
+
+def threebody_rhs(t, y, params):
+    y1, y2, y1_dot, y2_dot = y
+    if isinstance(params, float):
+        mu = params
```

Comment:
isnt this the moon mass or sth like this? If so, this might be a neat variable name. If not, keep the mu

Reply:
Yes, I mentioned this in the docstring. But I will rename the variable, as well